src/panels/ClusterPropertiesPanel.ts

```diff
@@ -64,6 +65,53 @@ export class ClusterPropertiesDataProvider implements PanelDataProvider<"cluster
         }
     }
 
+    private async _handleAbortClusterOperation(webview: MessageSink<ToWebViewMsgDef>) {
+        try {
+            const poller = await this.client.managedClusters.beginAbortLatestOperation(this.resourceGroup, this.clusterName);
+
+            poller.onProgress(state => {
+                if (state.status === "failed") {
+                    const errorMessage = state.error ? getErrorMessage(state.error) : "Unknown error";
+                    webview.postErrorNotification(errorMessage);
+                }
+            });
+
+            // Update the cluster properties now the operation has started.
+            await this._readAndPostClusterProperties(webview);
+
+            // Wait until operation completes.
+            await poller.pollUntilDone();
+        } catch (ex) {
+            const errorMessage = getErrorMessage(ex);
+            webview.postErrorNotification(errorMessage);
+        }
+    }
+
+    private async _handleReconcileClusterOperation(webview: MessageSink<ToWebViewMsgDef>) {
+        try {
+            const getClusterInfo = await this.client.managedClusters.get(this.resourceGroup, this.clusterName);
```

Comment:
Just a note for future reference because I don't think it's worth changing this right now, but ideally we'd pass `location` from the webview rather than performing an extra `get` here.

Maybe we can look at doing that as part of #291.